presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix scheduling issue in join spill to improve memory used by unspilled data.
+* Fix spilled query exceeding local memory limit due to incorrect memory size calculation when spilled data is read by SingleStreamSpiller.
+* Fix deadlock problem that results into join + limit getting stuck when spilling is enabled.
+* Improve RowExpressionFormatter output to be closer to valid SQL. Constant expressions are now output as ``TYPE 'value'`` (e.g. ``BIGINT'2'``) using TypeConstructor (with single quotes appearing in the string escaped with ``''``), with the exception of ``VARBINARY`` constants, which use the binary literal syntax (e.g. ``X'abc123'``). ``LIKE`` expressions are now output as valid SQL.
+* Add support for instance count in function :func:`array_position`.
```

Comment:
Link to documentation?

Reply:
The generated html will link to the documentation. See 
![image](https://user-images.githubusercontent.com/236188/124998602-8ff94800-e001-11eb-8e76-8b8a93050a09.png)
Hope that answers your question.

Reply:
`instance count` is not clear to me. Maybe

```
Add support to find the n-th instance in :func:`array_position`.
```

Reply:
done